### PR TITLE
doc: Add chassis for dev docs

### PR DESCRIPTION
**Pull request by inashivb**
Setup for Suricata Developer Guide

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/3344

### CODE REVIEW

File: doc/devguide/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,59 @@
+if HAVE_SURICATA_MAN
+man1_MANS = suricata.1
+endif
```
Comment (jasonish) [inline doc/devguide/Makefile.am]: Anything about the man page can be removed from the devguide section.  The PDF stuff can stay though.

File: doc/devguide/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Suricata Developer Guide
+
+This directory contains the Suricata Developer Guide. The
+[Sphinx Document Generate](http://sphinx-doc.org) is used to build the
```
Comment (jasonish) [inline doc/devguide/README.md]: `Generate` -> `Generator`. I realize this is a typo brought over from the user guide. Should probably fix it there as well (as a separate commit).

File: doc/devguide/convert.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
+#! /usr/bin/env python
```
Comment (jasonish) [inline doc/devguide/convert.py]: We can drop this file. It exists in the userguide still, but I don't think it useful anymore.

**Comment by inashivb** [comment]
Thanks, Jason! Replaced with #4650

**Pull request closed without merging**